src/inspector/unofficial.gni

```diff
@@ -72,4 +74,37 @@ template("inspector_gn_build") {
     outputs = [ "$node_gen_dir/src/{{source_name_part}}.json" ]
     args = [ "{{source}}" ] + rebase_path(outputs, root_build_dir)
   }
+
+  config("crdtp_config") {
+    include_dirs = [ protocol_tool_path ]
+  }
+
+  static_library("crdtp") {
```

Comment:
Should we add a `deps/inspector_protocol/unofficial.gni` for this library?

Reply:
`deps/inspector_protocol` actually has GN build files, but the revision rolled in Node somehow does not include the `crdtp` target: [BUILD.gn in main branch of Chromium](https://chromium.googlesource.com/deps/inspector_protocol.git/+/refs/heads/main/BUILD.gn) vs [the one shipped in Node](https://github.com/nodejs/node/blob/main/deps/inspector_protocol/BUILD.gn).

We can remove this target once the GN build file of `deps/inspector_protocol` includes it.

Reply:
`BUILD.gn` should be listed in https://github.com/nodejs/node/blob/532fff6b27be6b0d833d06b4a9fe46d6fb7f0f6c/tools/inspector_protocol/roll.py#L20.

Seems like it's been left out. Would you mind running `tools/inspector_protocol/roll.py` again? Or maybe I can do it.

Reply:
I ran the script and it successfully updated the `BUILD.gn` to the one that has the `crdtp` target, however the file has a dependency of `//base` which is part of Chromium so we can not use it to build Node.

I think we have to put the `crdtp` target in `src/inspector/unofficial.gni` for now, `deps/inspector_protocol` has its own GN build file and we should not overwrite it. I can try to make an upstream change to make inspector_protocol's GN build file compatible with `build_with_chromium = false`.

Reply:
sounds good, thank you!